### PR TITLE
Clean up BERT, RoBERTa doc-strings

### PR DESCRIPTION
**Pull request by abheesht17**
@jbischof

### CODE REVIEW

File: keras_nlp/models/roberta.py [hunk 1]
```diff
@@ -30,32 +30,32 @@ class RobertaCustom(keras.Model):
 
     This network implements a bi-directional Transformer-based encoder as
     described in ["RoBERTa: A Robustly Optimized BERT Pretraining Approach"](https://arxiv.org/abs/1907.11692).
-    It includes the embedding lookups and transformer layers, but not include
-    the masked language model network.
+    It includes the embedding lookups and transformer layers, but not the masked
+    language model or classification task networks.
```
Comment (mattdangerw) [inline keras_nlp/models/roberta.py:34]: Maybe keep the original, but say...

but does not include the masked language model head used during pretraining.

Unlike BERT there is no classification head used in pretraining, so the sentence you have is a bit confusing.

**Review by mattdangerw — APPROVED** [review]
lgtm! just one nit

**Comment by jbischof** [comment]
I am asking about style for typehints in this morning Keras meeting if you're OK waiting a little bit. Doctring typehints are very inconsistent in the core Keras package.

**Comment by abheesht17** [comment]
> I am asking about style for typehints in this morning Keras meeting if you're OK waiting a little bit. Docstring typehints are very inconsistent in the core Keras package.

Sure!

**Review by jbischof — APPROVED** [review]
There wasn't a clear conclusion yet but your version is closer to the current proposal of using the python types in backticks. LGTM

**Comment by abheesht17** [comment]
> There wasn't a clear conclusion yet but your version is closer to the current proposal of using the python types in backticks. LGTM

Thanks!
